internal/testcontext/context.go

```diff
@@ -120,11 +123,18 @@ func (ctx *Context) Dir(subs ...string) string {
 	})
 
 	dir := filepath.Join(append([]string{ctx.directory}, subs...)...)
-	_ = os.MkdirAll(dir, 0744)
+	err := os.MkdirAll(dir, 0744)
+	if err != nil {
+		ctx.test.Fatal(err)
+	}
 	return dir
 }
 
-// File returns a filepath inside temp
+// File returns a filepath inside temp.
+//
+// subs are joined into a single path adding a separator if necessary, being the
+// last one the name of the file. Directories are created, the file isn't
+// created.
 func (ctx *Context) File(subs ...string) string {
```

Comment:
I wonder whether subs should be renamed to elem to match https://golang.org/pkg/path/filepath/#Join declaration.

Reply:
I wondered the same, but I didn't want to change things that I thought that they were expressly necessary.

If you think that's better, we can change it in this same branch.

Reply:
Yeah, change. If a commit changes less than 100 lines, then don't worry too much about putting other cleanup code in the same PR. Separating them out will cause both more dev and review time.

Reply:
I didn't do it because of the number of changes, it was mostly because I wanted to do the changes that I believed that were objective and `subs` replaced by `elem` I didn't think that such reason was a strong for arguing the change rather than being a change of subjective taste.

Because it bubbled from you and you are the author, so good to do it.